indexers/vector/PysparnnIndexer/__init__.py

```diff
@@ -158,6 +158,7 @@ def query(self, vectors: 'SparseMatrixType', top_k: int, *args, **kwargs):
         distances = []
         indices = []
         for record in index_distance_pairs:
+            sorted_record = sorted(record, key=lambda x: float(x[0]),reverse=False)
```

Comment:
this sorted record should be used after in the zip

Reply:
yes, commited too fast, fixed.